druid/src/widget/label.rs

```diff
@@ -310,8 +316,8 @@ impl<T: Data> Label<T> {
 impl<T> Dynamic<T> {
     fn resolve(&mut self, data: &T, env: &Env) -> bool {
         let new = (self.f)(data, env);
-        let changed = new != self.resolved;
-        self.resolved = new;
+        let changed = new.as_str() != &*self.resolved;
```

Comment:
I usually prefer `as_ref` over `&*`
```suggestion
        let changed = new.as_str() != self.resolved.as_ref();
```

Reply:
ah totally if `AsRef` is available, I thought this was only available via deref. :)